key.go

```diff
@@ -124,3 +171,17 @@ const (
 	// AltModifier represents either alt keys being held
 	AltModifier
 )
+
+// KeyShortcut reprents the key combination associable to a command (i.e. copy, paste...)
+type KeyShortcut int
+
+const (
+	// ShortcutNone represents the key combinations that do not have command associated to
+	ShortcutNone KeyShortcut = 1 << iota
```

Comment:
I think that ShortcutNone would be better as `iota` rather than `1 << iota`?
Or is the intention that shortcuts could be compounded?

Reply:
Fixed in 3929f54